aleth-key/KeyAux.h

```diff
@@ -1,21 +1,8 @@
 #pragma once
 
-/*
-	This file is part of cpp-ethereum.
-
-	cpp-ethereum is free software: you can redistribute it and/or modify
-	it under the terms of the GNU General Public License as published by
-	the Free Software Foundation, either version 3 of the License, or
-	(at your option) any later version.
-
-	cpp-ethereum is distributed in the hope that it will be useful,
-	but WITHOUT ANY WARRANTY; without even the implied warranty of
-	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-	GNU General Public License for more details.
-
-	You should have received a copy of the GNU General Public License
-	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
-*/
+// Aleth: Ethereum C++ client, tools and libraries.
+// Copyright 2019 Aleth Authors.
+// Licensed under the GNU General Public License, Version 3.
 /** @file KeyAux.cpp
  * @author Gav Wood <i@gavwood.com>
  * @date 2014
```

Comment:
The `@date` comments should be also removed. If possible you can add the year from here to the top comment as `Copyright 2014-2019 Aleth Authors`.

Reply:
@gumb0 / @chfast Do we want to include a year there at all since that presumably means we need to update it regularly? Or do we need to include the current year (license end date) in order for the license message to be considered valid?

Reply:
The current year is enough. It sets the year until the copyright holds. The "start" year is purely informational and should not be added in this case, because do not match the year when a file was created. Please make this change and it should be ready for final review.